app/views/issues/index.html.erb

```diff
@@ -60,6 +61,13 @@
           <td class="text-nowrap"><%= link_to t(".reports_count", :count => issue.reports_count), issue %></td>
           <td><%= link_to reportable_title(issue.reportable), reportable_url(issue.reportable) %></td>
           <td><%= link_to issue.reported_user.display_name, issue.reported_user if issue.reported_user %></td>
+          <td>
+            <% unique_reporters = issue.reports.uniq { |report| report.user.id } %>
+            <% unique_reporters[..2].each_with_index do |report, index| %>
+              <%= link_to(report.user.display_name, report.user) %><%= "," unless index == 2 || index == unique_reporters.size - 1 %>
+            <% end %>
+            <%= "..." if unique_reporters.size > 3 %>
```

Comment:
Maybe instead of truncating the list here we should put the whole thing in but use CSS to limit the width and add an ellipsis? With a tooltip to show the follow value even?